### DISCUSSION TITLE
Help with error when try to update/upgrade

### DISCUSSION THREAD

**Discussion by DariBer** [ajfriesen/ESPHome-AirGradient#8] [Q&A] [answered]
I flashed my AirGardient first with USB Carle to a computer and then used Chrome. Now when I want to update/upgrade from ESPHome from Home Assistant and use the wireless option, then I get this error message. 

"ERROR Error binary size: Error: ESP has been flashed with wrong flash size. Please choose the correct 'board' option (esp01_1m always works) and then flash over USB."

I haven't change anything in the config since I flashed it for the first time. So not sure why I get this error.

**Comment by ajfriesen** [comment, marked as answer]
I think this looks like a pretty generic ESPHome question, therefore I would suggest asking in the Home Assistant forum: https://community.home-assistant.io/c/esphome/36

I just updated without any problem over WiFi.
